bundles/org.openhab.binding.bmwconnecteddrive/README.md

```diff
@@ -0,0 +1,862 @@
+| Rendered Image of the Vehicle | image            | png                 | Image  |
+| Image Viewport                | image            | view                | String |
+| Image Picture Size            | image            | size                | Number |
+
+
+#### Troubleshooting
+
+Available for all Vehicles!
+Switch will log a *Vehicle Data Fingerprint* into the openHAB log.
+Please check [TroubleShooting Section](#TroubleShooting) for further advice.
+
+| Channel Label                       | Channel Group ID | Channel ID          | Type   | Description                                       |
+|-------------------------------------|------------------|---------------------|--------|---------------------------------------------------|
+| Log Vehicle Fingerprint             | troubleshoot     | vehicle-fingerprint | Switch | Forces log entries in openHAB logger in order to raise issues |
+
+## Further Descriptions
+
+### List Interface
+
+Currently there are 3 occurrences of dynamic data delivered as Lists
+
+* Upcoming Services
+* Check Control Messages
+* Last Destinations
+
+In order to provide all data the lists are organized the following way based on the Service Group example
+
+* **size** is the total length of the list. Read-only.
+* **index** is the current selected index starting with 0 => list with size 10, valid index is between 0 - 9. If the index is set to _out of bounds_ it's set to first element. Read-write.
+* **next** declares a switch for scrolling forward through the list in an endless way. Reaching end of list will start again from 0. Read-write.
+* **name**, **mileage** and **date** declares the list item. It will change when setting new index or perform next. Read-only.
+
+### Range vs Range Radius
+
+<img align="right" src="./doc/range-radius.png" width="400" height="350"/>
+
+ will observe differences in the Vehicle Range and Range Radius values. 
+While Range is indicating the possible distance to be driven on roads the range radius indicates the reachable range on the Map.
+
+The right picture shows the distance between Kassel and Frankfurt in Germany. 
+While the Air-line Distance is ~145 Kilometer the Route Distance is ~192 Kilometer.
+So Range value is the normal remaining range.
+See the Section [OpenStreetMap](#openstreetMap-widget) how the Range Radius is used to indicate the reachable Range on Map.
+Please note this is just an indicator and the effective range, especially for Electric Vehicles, 
+depends on many factors like driving style and electric consumers. 
+ 
+### Last Status Update Timestamp
+
+A timestamp is showing the last Vehicle Status update. If this isn't shown correctly please check the date settings.
+In case of Raspberry Pi execute *raspi-config*, select *Localization Options*, the *Change Time Zone*
+Select your *Geographical Area* and afterwards the correct City.
+One restart of openHAB service with *systemctl restart openhab2* is necessary in order to see the corrected Time Settings.
```

Comment:
Did you take into account, that the timezone can be configured within the framework, too? E.g. via PaperUI. You can retrieve the configured timezone by injecting this class: https://www.openhab.org/javadoc/v2.5/org/eclipse/smarthome/core/i18n/timezoneprovider

Reply:
No, to be honest I didn't checked that. I added the code and you're right it's showing the same results. With this in mind I also deleted the section in readme. I think the topic is too generic to mention it in this specific binding.